### ISSUE TITLE
ampl_function_demo/src missing from conda package

### ISSUE THREAD

**Issue by joshua-cogliati-inl** [Pyomo/pyomo#2345]
## Summary

ampl_function_demo/src is missing from the conda package for pyomo 6.3.0

### Steps to reproduce the issue

```console
$ pyomo build-extensions
...
```

### Error Message

```
**** Building PyNumero libraries ****
-- The C compiler identification is AppleClang 12.0.0.12000032
-- The CXX compiler identification is AppleClang 12.0.0.12000032
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeLists.txt:18 (INCLUDE):
  INCLUDE could not find requested file:
    /Users/fred/miniconda3/envs/raven_libraries/lib/python3.7/site-packages/pyomo/contrib/pynumero/src/../../ampl_function_demo/src/FindASL.cmake
```

### Information on your system

Pyomo version: 6.3.0
Python version: 3.7
Operating system: MacOS
How Pyomo was installed (PyPI, conda, source): conda

```
$ tar -tvjf pyomo-6.3.0-py37h0582d14_1.tar.bz2 | grep 'ampl_function_demo'
-rw-rw-r--  0 501    20        159 Mar  2 15:43 lib/python3.7/site-packages/pyomo/contrib/ampl_function_demo/__pycache__/__init__.cpython-37.pyc
-rw-rw-r--  0 501    20        165 Mar  2 15:43 lib/python3.7/site-packages/pyomo/contrib/ampl_function_demo/tests/__pycache__/__init__.cpython-37.pyc
-rw-rw-r--  0 501    20        461 Mar  2 15:43 lib/python3.7/site-packages/pyomo/contrib/ampl_function_demo/__pycache__/plugins.cpython-37.pyc
-rw-rw-r--  0 501    20        934 Mar  2 15:43 lib/python3.7/site-packages/pyomo/contrib/ampl_function_demo/__pycache__/build.cpython-37.pyc
-rw-rw-r--  0 501    20       2091 Mar  2 15:43 lib/python3.7/site-packages/pyomo/contrib/ampl_function_demo/tests/__pycache__/test_ampl_function_demo.cpython-37.pyc
-rw-rw-r--  0 501    20        761 Feb 23 17:12 lib/python3.7/site-packages/pyomo/contrib/ampl_function_demo/plugins.py
-rw-rw-r--  0 501    20       1118 Feb 23 17:12 lib/python3.7/site-packages/pyomo/contrib/ampl_function_demo/build.py
-rw-rw-r--  0 501    20       2385 Feb 23 17:12 lib/python3.7/site-packages/pyomo/contrib/ampl_function_demo/tests/test_ampl_function_demo.py
-rw-rw-r--  0 501    20          0 Feb 23 17:12 lib/python3.7/site-packages/pyomo/contrib/ampl_function_demo/tests/__init__.py
-rw-rw-r--  0 501    20          0 Feb 23 17:12 lib/python3.7/site-packages/pyomo/contrib/ampl_function_demo/__init__.py
```

**Comment by michaelbynum** [comment]
Pyomo 6.4.0 will be released shortly, which will fix the problem.

**Comment by jsiirola** [comment]
Duplicate of #2317, resolved in #2319, released in 6.4.0.
